web/src/beta/lib/core/engines/Cesium/hooks.ts

```diff
@@ -600,6 +609,17 @@ export default ({
     [engineAPI],
   );
 
+  useEffect(() => {
+    if (!cesium.current?.cesiumElement) return;
+    const canCameraMove = !!(featureFlags & MAP_FEATURE_FLAGS.CAMERA_MOVE);
```

Comment:
`canXxxXxx` is fine. 
But maybe `allowXxxXxxxx`, `xxxXxxxAllowed` or `xxxXxxxEnabled` is more accurate?

Reply:
Missed this comment

Reply:
Fixed: [Update hooks.ts](https://github.com/reearth/reearth/pull/507/commits/639bfba18f19f27f407e2bb81f79e308495a0039)